### PR TITLE
feat(navigationButtons): add NavigatorButtons component

### PR DESCRIPTION
**Pull request by maximilianoforlenza**
feat(navigator): add navigator buttons

### CODE REVIEW

File: src/components/NavigationButtons/index.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+                color="#333"
+                title={buttonTitle}
+                onPress={save}
+            />
+        </Col>
+    </Row>
+);
+
+NavigatorButtons.propTypes = {
+    back: PropTypes.func.isRequired,
```
Comment (navarroaxel) [inline src/components/NavigationButtons/index.js]: `onBack`

File: src/components/NavigationButtons/index.js [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+    save: PropTypes.func.isRequired,
```
Comment (navarroaxel) [inline src/components/NavigationButtons/index.js]: `onSubmit`.

File: src/components/NavigationButtons/index.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+                title="Anterior"
+                onPress={back}
+            />
+        </Col>
+        <Col>
+            <Button
+                icon={styles.icon}
+                buttonStyle={styles.buttonStyle}
+                color="#333"
+                title={buttonTitle}
```
Comment (navarroaxel) [inline src/components/NavigationButtons/index.js]: `submitButtonText`

File: src/components/NavigationButtons/index.js [hunk 4]
```diff
@@ -0,0 +1,37 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import {Button} from 'react-native-elements';
+import {Col, Row} from '@indec/react-native-commons';
```
Comment (navarroaxel) [inline src/components/NavigationButtons/index.js]: usa el Button de este package.

**Comment by maximilianoforlenza** [comment]
listo

**Pull request merged**